certified-connectors/personr/apiDefinition.swagger.json

```diff
@@ -0,0 +1,506 @@
+                                        },
+                                        "tags": {
+                                            "type": "array",
+                                            "items": {},
+                                            "description": "tags"
+                                        },
+                                        "flowName": {
+                                            "type": "string",
+                                            "description": "flowName"
+                                        },
+                                        "status": {
+                                            "type": "string",
+                                            "description": "status"
+                                        }
+                                    },
+                                    "description": "response"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/api-request-applicant-check": {
+            "post": {
+                "responses": {
+                    "default": {
+                        "description": "default",
+                        "schema": {}
+                    }
+                },
+                "summary": "Request Applicant Check",
+                "description": "Confirm all documents are uploaded, and start the check.",
+                "operationId": "Api-request-applicant-check",
+                "parameters": [
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "required": false,
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "applicantId": {
+                                    "type": "string",
+                                    "description": "applicantId",
+                                    "title": "Applicant ID"
+                                }
+                            }
+                        }
+                    }
+                ]
+            }
+        }
+    },
+    "definitions": {},
+    "parameters": {},
+    "responses": {},
+    "securityDefinitions": {
+        "Auth. Token": {
+            "type": "apiKey",
+            "in": "header",
+            "name": "Authorization"
+        }
+    },
+    "security": [
+        {
+            "Auth. Token": []
+        }
+    ],
+    "tags": [],
+    "x-ms-connector-metadata": [
+        {
+            "propertyName": "Website",
+            "propertyValue": "https://personr.co"
+        },
+        {
+            "propertyName": "Privacy policy",
+            "propertyValue": "https://personr.co/legals"
+        },
+        {
+            "propertyName": "Categories",
+            "propertyValue": "Identity Verification"
```

Comment:
Hello @charliewesterman, 

Only below categories are available, please update.  Please refer this URL - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json#:~:text=%22x%2Dms%2Dconnector,%5D

[AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website].